### DISCUSSION TITLE
Migrating from Docker to repository

### DISCUSSION THREAD

**Discussion by lordratner** [lldap/lldap#1034] [Q&A]
Hi!

I run lldap on Proxmox, and I'd like to transition to the repository version to reduce the resource usage by removing docker. 

How can I migrate my current users/groups from the docker 0.6 version to the repository 0.6 version?

Thanks!
Seth

**Comment by nitnelave** [comment]
Aside from updating your config for the correct relative paths of everything, you just need to bring your database along. If it was already a mounted SQLite folder, for instance, just point the config to that folder. If it was in a volume, copy the `users.db*` out of the volume. If using an external DB like PostgreSQL, point to the PG instance (so, you probably don't have to change anything, unless you're also migrating PG out of docker)
